fetch.bs

```diff
@@ -4974,8 +5192,13 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
       <p>If <var>aborted</var> is set, then:
 
       <ol>
+       <li><p>Call <a for=/>handle response end</a> with <var>fetchParams</var> and <var>response</var>.
```

Comment:
@annevk: ended up reporting the resource-timing the moment `response end time` is set, which could be either when the response is aborted or when the last byte of the body was sent.

Reply:
It seems weird that this is invoked and then below response end time is set. And why is this only when the response is aborted and not when it errored? (Note that "if aborted" and "aborted" are different things.)

Reply:
Also, response end time isn't set for a successful response?

Reply:
It is, in `handle response end`. The one here is actually a duplicate.